src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1311,35 +1311,34 @@ protected override void GetItem(string path)
                     // If we want to retrieve the file streams, retrieve them.
                     if (retrieveStreams)
                     {
-                        if (!isContainer)
+                        foreach (string desiredStream in dynamicParameters.Stream)
                         {
-                            foreach (string desiredStream in dynamicParameters.Stream)
+                            // See that it matches the name specified
+                            WildcardPattern p = WildcardPattern.Get(desiredStream, WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant);
+                            bool foundStream = false;
+
+                            foreach (AlternateStreamData stream in AlternateDataStreamUtilities.GetStreams(result.FullName))
                             {
-                                // See that it matches the name specified
-                                WildcardPattern p = WildcardPattern.Get(desiredStream, WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant);
-                                bool foundStream = false;
+                                if (!p.IsMatch(stream.Stream))
+                                { continue; }
```

Comment:
Please either keep the style as-is or fully hang the if-block

Reply:
It actually is keeping the style as it was. (I moved this block a bit earlier in the file to do streamname processing before deciding whether it was a container or data, to decide what to do with it.)

I'm willing to fully hang the if-block, though, because this style annoys me too.

Reply:
Ah, understood.

Yeah, if you'd like to make that change, that would be appreciated

Reply:
This has been changed in my latest push.